modules/lambda@edge/main.tf

```diff
@@ -78,6 +78,8 @@ resource "aws_lambda_function" "default" {
   function_name    = module.function_label[each.key].id
   runtime          = each.value.runtime
   handler          = each.value.handler
+  memory_size      = each.value.memory_size != null ? each.value.memory_size : 128
```

Comment:
@mihaiplesa I just noticed these lines. Wouldn't it be better to set the default in the variables? i.e. `optional(number, 128)`. Lambda requires these values to be set so defaulting them should be fine for existing module users.